lib/oban/worker.ex

```diff
@@ -173,6 +173,9 @@ defmodule Oban.Worker do
   @doc """
   The `perform/2` function is called when the job is executed.
 
+  The map given as the first argument will have string keys, even if atom keys were used when
```

Comment:
As mentioned above, this is only true when Oban is executing jobs, not if you call `MyApp.Worker.perform/2` directly.